src/components/markdown-input.js

```diff
@@ -17,7 +17,7 @@ import GithubFlavoredMarkdown from '../common/github-flavored-markdown.js';
 import ResourceSelectorDialog from './resource-selector/resource-selector-dialog.js';
 
 function MarkdownInput({ minRows, disabled, inline, debounced, renderAnchors, sanitizeCdnUrls, value, onBlur, onChange, preview, embeddable, maxLength, ...rest }) {
-  const { locations } = useStorage();
+  const storage = useStorage();
```

Comment:
I've decided to keep the `storage` naming versus `roomStorage` (and `roomStorageContext` etc.) because we use `storagePlan` and not `roomStoragePlan` and it is clearer to correlate them.

So I just changed the model of `storage` from containing `locations` to directly containing what was the room storage locaiton.

Reply:
:+1: